backend/app/Models/Concerns/DetectsStaleData.php

```
@@ -0,0 +1,30 @@
+ *
+ * @property \Illuminate\Support\Carbon|null $updated_at
+ */
+trait DetectsStaleData
+{
+    /**
+     * データベース上のデータが、自分が取得した後に更新されていないか確認する。
+     */
+    public function hasBeenUpdatedSinceRetrieval(string $fetchedAt): bool
+    {
+        try {
+            $fetchedAtTime = Carbon::parse($fetchedAt);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Carbon' in method 'hasBeenUpdatedSinceRetrieval'. (undefined)

(StaticAccess)